### PR TITLE
remove unnecessary UDFs in microsalt sample BE (#3886)(patch)

### PR DESCRIPTION
**Pull request by diitaz93**
## Description
As requested in https://github.com/Clinical-Genomics/clinical-genomics-ui/issues/545, we are removing `quantity` and `concentration_sample` from MicrobialSample model in `cg/models/orders/samples.py`

### Changed

- Microbial Sample
- Fixtures from Microsalt and SARS-Cov2

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b remove-attributes-microbial-sample] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Review by Vince-janv — COMMENTED** [review]
Could you have a look at where these fields are propagated and remove them there as well? For example the file mentioned here:  https://github.com/Clinical-Genomics/clinical-genomics-ui/issues/545#issuecomment-2419397886

**Comment by islean** [comment]
> Could you have a look at where these fields are propagated and remove them there as well? For example the file mentioned here: [Clinical-Genomics/clinical-genomics-ui#545 (comment)](https://github.com/Clinical-Genomics/clinical-genomics-ui/issues/545#issuecomment-2419397886)

My impression is that the LIMS parsing is not workflow specific, it just uses a dump of the Sample model. If so, removing the LIMS UDFs would disable the propagation of the values for every workflow. Would we still want to remove them, should my impression be correct?

**Comment by Vince-janv** [comment]
> My impression is that the LIMS parsing is not workflow specific, it just uses a dump of the Sample model. If so, removing the LIMS UDFs would disable the propagation of the values for every workflow. Would we still want to remove them, should my impression be correct?

Yeah, I meant: remove them if they where unique to the microbial samples 👍

**Comment by diitaz93** [comment]
> > My impression is that the LIMS parsing is not workflow specific, it just uses a dump of the Sample model. If so, removing the LIMS UDFs would disable the propagation of the values for every workflow. Would we still want to remove them, should my impression be correct?
> 
> Yeah, I meant: remove them if they where unique to the microbial samples 👍

My judgement is that there is nothing microbial-specific in that module to remove

**Comment by diitaz93** [comment]
## Testing
Together with https://github.com/Clinical-Genomics/clinical-genomics-ui/pull/563

File: cg/models/orders/samples.py [hunk 1]
```diff
@@ -265,20 +265,9 @@ class MicrobialSample(OrderInSample):
     well_position: str | None
     # "Required if "Other" is chosen in column "Species""
     organism_other: constr(max_length=Organism.internal_id.property.columns[0].type.length) | None
-    # "These fields are not required"
-    concentration_sample: float | None
-    quantity: int | None
```
Comment (islean) [inline cg/models/orders/samples.py:270]: This will affect covid orders as well - is that planned?

**Review by ChrOertlin — APPROVED** [review]

File: cg/models/orders/samples.py (hunk 1 above)
Comment (diitaz93) [reply cg/models/orders/samples.py:270]: No it will not. We tested it with a sars order too

**Pull request merged**